docs/connections/oracle-target.md

```diff
@@ -0,0 +1,100 @@
+---
+title: OracleDB Connector
+---
+
+Macrometa collection connectors allow you to extract data from or send data to an existing OracleDB collection.
+
+## Prerequisites
+
+- Macrometa account with admin permissions.
+- Remotely accessible, running OracleDB instance. Make sure to have the required permissions on the database, schema, and tables that you want to replicate from Oracle.
+- Source collections must not be empty.
+- Access to `V$DATABASE` and `V_$THREAD` performance views. These are required to verify setting configuration while setting up your Oracle database and to retrieve the database's Oracle System ID.
+
+## OracleDB Source
+
+Read this section carefully before you set up your OracleDB source connector.
+
+### Multi-Tenant
+
+If **Multi-Tenant** is set as **True** and a **Pluggable Database Name** is not provided, then CDB$ROOT container is used to access the database.
+
+### Log-Based Replication Requirements
```

Comment:
@diana-macrometa 
This section seems to be coming from oracle source. 
@koshy-thomas Do we need log based replication enabled for oracle target?

Reply:
@praminda-macrometa no we don't need log based replication anabled

Reply:
That section refers to the OracleDB source, as indicated by the heading on line 14. I added Oracle target info at the bottom, starting at line 98.

Reply:
The file name does not matter. Once 17.16 is merged, I'll move the contents of this file to the correct one in the Connector Types section and update the lists where applicable.

Reply:
Any other concerns, or can I merge this?